README.md

```diff
@@ -54,9 +41,12 @@ To run browser tests locally make sure `storybook-vue` is up and running.
 docker-compose -f docker-compose.yml -f docker-compose.browsertests.yml up browsertests
 ```
 
+Note: If you were not already running the main docker-compose services your tests may fail their first run as things are still being built.
+
 ##### On the host machine
 
-Navigate to `vue-components` subdirectory and run it with your storybook URL specified, e.g.
+Navigate to `vue-components` subdirectory and run it with your storybook URL specified:
+
 ```sh
 STORYBOOK_URL=localhost:6005 npm run e2e
```

Comment:
This command did not used docker-compose before this change so I am leaving it as it is.